SoftLayer/managers/registration.py

```diff
@@ -36,6 +36,92 @@ def detail(self, identifier):
             return self.registration.getObject(mask=mask, id=registration['id'])
         return None
 
+    def update_all(self, contact_id, skip_subnets=None):
+        """Updates every subnet to contact
+
+        :param list skip_subnets: A list of subnets ids to skip
+        :param int contact_id: Id of a Account_Regional_Registry_Detail that is a person
+        :return:A list of subnet registrations
+        """
+        skip_subnets = [] if skip_subnets is None else skip_subnets
```

Comment:
instead of doing this `skip_subnets = [] if skip_subnets is None else skip_subnets`, which is hard to read, just set `skip_subnets = []` in the function declaration.

Reply:
done